package.json

```diff
@@ -35,10 +35,10 @@
     "develop": "webpack --mode=development --watch & webpack-dev-server --mode=development & wait",
     "build:proto": "./scripts/compile_proto.sh",
     "build:webpack": "webpack --mode='production'",
-    "build:documentation": "typedoc",
+    "build:documentation": "arkit src/ -o ./media/dependencies.png && typedoc --media ./media --gitRevision master --mode modules",
```

Comment:
Once docs generation is completely removed as in typescript-demo-lib, then I believe even these things would be done in the same way. So the images will need to be copied to the `gh-pages` branch.

Reply:
yeah I agree, this `build:documentation` is just here for when that happens, I've just taken it out of the `npm run build:all` command.